protocol/0081-SUCM-successor_markets.md

```diff
@@ -70,7 +71,32 @@ Two proposals that name the same parent can be submitted. Both can be approved b
 
 A new market proposal sets parent market Id to a market that has settled. The parent market has non-zero insurance pool balance. If the new market clears the opening auction before `parent settlement time + market.value.windowLength` then the virtual stakes are carried over and the relevant fraction of the insurance pool is transferred over (<a name="0081-SUCM-006" href="#0081-SUCM-006">0081-SUCM-006</a>).
 
-A new market proposal sets parent market Id to a market that has settled. The parent market has non-zero insurance pool balance. If the new market clears the opening auction after `parent settlement time + market.value.windowLength` then no virtual stakes are carried over, there is no transfer into the insurance pool of the new market from the parent and the new market has no parent market Id set (<a name="0081-SUCM-007" href="#0081-SUCM-007">0081-SUCM-007</a>).
+A new market proposal sets parent market Id to a market that has settled. The parent market has non-zero insurance pool balance. If the new market clears the opening auction after `parent settlement time + market.value.windowLength` then no virtual stakes are carried over, there is no transfer into the insurance pool of the new market from the parent and the new market has no parent market Id set (<a name="0081-SUCM-007" href="#0081-SUCM-007">0081-SUCM-007</a>)
+
+Successor markets can't be enacted if the parent market is still in the "proposed" state (<a name="0081-SUCM-008" href="#0081-SUCM-008">0081-SUCM-008</a>)
+
+Successor markets can be enacted when the parent market is in opening auction (<a name="0081-SUCM-009" href="#0081-SUCM-009">0081-SUCM-009</a>)
+
+A successor market proposal can be enacted when the parent market is in one of the following states: Suspended, Active, Trading terminated or Settled (settled within the successor time window) (<a name="0081-SUCM-010" href="#0081-SUCM-010">0081-SUCM-010</a>)
```

Comment:
Yes, but it doesn't mean that a successor market proposal cannot be enacted when the parent market is in other states. E.g. 
- parent Settled and past the successor time window... fine but pointless no virtual stake to copy, no insurance pool balance to transfer. 
- proposed, see above.

Reply:
IIRC, there's some ACs covering the settled+ past successor window elsewhere. I think the different states mentioned here are more about explicitly stating that successors can get enacted when parent is in one of these states, without issues

Reply:
- I thought we had one for when the time window has already expired... We definitely should.
- - 0081-SUCM-007 covers what happens if it expires during the child proposal
- - We probably need another one to cover it already having expired prior to the child proposal: "Propose a successor market which specifies a parent which is settled, and for which the successor time window has expired". Expect pass/fail?
- I think Proposed is covered in 0081-SUCM-008

Reply:
@jgsbennett I've added a suggestion below to cover the middle bullet point. https://github.com/vegaprotocol/specs/pull/1754/files#r1226491529